app/src/main/java/com/github/polybooks/MainActivity.kt

```diff
@@ -24,7 +24,9 @@ class MainActivity : AppCompatActivity() {
 
         val dbButton: Button = findViewById(R.id.button_open_db_tests)

```

Comment:
The button probably deserves to be renamed?
Also again no point in keeping the 2 lines commented

Reply:
This comes from the fact that the link between the said button and the books is not definitive (will we display books or sale)? But you are right that it won't be simply a "database" anymore, so I replace with a more meaningful name